### ISSUE TITLE
Initial Release?

### ISSUE THREAD

**Issue by eezstreet** [JACoders/OpenJK#186]
What all is left on the initial release?

**Comment by mrwonko** [comment]
> Why was asking about a release an issue? xD Close?

No, this is actually a good place to discuss what should go into the first release, I think. So, what's left?

I think the most important thing is some form of automatic updater; otherwise people will inevitably stay on old releases. I'm inclined to try writing one myself...

**Comment by BruceJohnJennerLawso** [comment]
@mrwonko, about that automatic updater, were you thinking something along the lines of a script? I could bang together a python script for that if its what you had in mind.

**Comment by ensiform** [comment]
A python script would be not very good for Windows though as not everyone has it installed.

**Comment by Razish** [comment]
If we're going with scripts, we could take inspiration from Xonotic's update mechanism.
